extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -131,7 +131,11 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     if (this.plpContainer_.querySelector('.i-amphtml-amp-story-shopping-plp')) {
       return;
     }
-    const shoppingData = this.storeService_.get(StateProperty.SHOPPING_DATA);
+
+    const pageElement = this.element.parentElement;
```

Comment:
This is unsafe since we aren't explicitly getting a reference to the amp-story-page element.
It's unlikely but possible that the dom structure could change. 
[This helper](https://github.com/ampproject/amphtml/blob/main/extensions/amp-story-page-attachment/0.1/amp-story-page-attachment.js#L82) can be used as an example.